hands-on-exercise/4-run-in-prod/4-migration-leaderboard.md

```diff
@@ -278,20 +278,20 @@ func ComputeInitGenesis(ctx sdk.Context, playerInfosk types.PlayerInfoKeeper) (*
 
 <HighlightBox type="best-practice">
 
-To further limit the dependency of the leaderboard module on the the checkers module, you could consider:
+To further limit the dependency of the leaderboard module on the checkers module, you could consider to options:
```

Comment:
Did you mean "2 options"? Because these are not options, but 2 steps to do in order to "limit the dependency".

Reply:
Sorry, I don't know where "options" came from here. I think I meant: 

You could consider to: 
* Have...
* Keep...

But that's actually worse than not using "to" and the bullets starting with "Having" and "Keeping". I'll change it.